doc-content/enterprise-only/getting-started/business-process-proc.adoc

```diff
@@ -26,77 +26,95 @@ The *Package* specifies the location inside the existing project where the asset
 +
 [source,java]
 ----
 System.out.println(application.getProperty());
 ----
-+
-. Scroll down and expand the *Data Assignments* section and click image:getting-started/btn_assign.png[] next to *Assignments*.
-. In the *Validation Data I/O* window, click *Add* to create the following assignments:
+.. Expand *Data Assignments* and click image:getting-started/btn_assign.png[] next to *Assignments*.
+.. In the *Validation Data I/O* window, click *Add* and create the following assignments:
 +
 image::getting-started/val-data-io.png[]
 
-. Click *Save*.
-. In the diagram editor, click *Save*, and *Save*, to confirm your changes.
+.. Click *Save*.
+. In the diagram editor, click *Save* and click *Save* again to confirm your changes.
 
 == Defining the validation data
-This section describes how to define the data that determines whether the application data is correct or contains an error or any missing information.
+This section shows you how to define the validation data that determines whether the application data is correct, contains an error, or has missing information.
 
 .Procedure
-. Click on the *Validation* task and create an outgoing connection to an exclusive gateway.
+
+. Create an outgoing connection from the *Validation* task to an exclusive gateway:
+.. Click the *Validation* task and click the *Create Parallel* icon.
+.. Hover over the image:getting-started/config-icon.png[] icon of the parallel and click the *Convert into Exclusive* icon.
 +
 image::getting-started/exclusive_gateway.png[]
 
-. Create an outgoing connection from the exclusive gateway to a new business rule task.
+. Create an outgoing connection from the exclusive gateway to a new business rule task:
+.. Click the exclusive gateway and click the *Create Task* icon.
+.. Hover over the image:getting-started/config-icon.png[] icon of the task and click the *Convert into Business Rule* icon.
 +
 image::getting-started/new_br_task.png[]
+.. If the *Diagram properties* panel is not open, in the upper-right corner click the *Diagram properties* image:getting-started/diagram_properties.png[] icon.
+.. In the *Diagram properties* panel, enter `Retract Validation` in the *Name* field.
+.. Expand *Implementation/Execution* and enter `error` in the *Rule Flow Group* field.
 
-. Click the created connection.
+. Configure the connection between the exclusive gateway and the business rule task:
+.. Click the connection.
 +
 image::getting-started/created-connection.png[]
+.. If the *Diagram properties* panel is not open, in the upper-right corner click the *Diagram properties* image:getting-started/diagram_properties.png[] icon.
+.. In the *Diagram properties* panel, enter `Invalid` in the *Name* field.
+.. Expand *Implementation/Execution* and select *Expression* in the *Condition Expression* section.
+.. From the drop-down list, select *drools* and enter `ValidationErrorDO()` in the *Condition Expression* field.
```

Comment:
What is the name of the list? According to IBM SG p209 "Refer to a drop-down list by its label, followed by list." So we should be saying "From the <apple> list, select...."

Reply:
True, there is no name, adopted suggestion